cli.js

```diff
@@ -56,10 +74,44 @@ function run(file) {
 			return;
 		}
 
```

Comment:
It can both return a promise and emit events. Events are emitted on the object, not promise.